src/redis.go

```diff
@@ -37,14 +41,19 @@ func main() {
 
 	rawMetrics, rawKeyspaceMetrics, metricsErr := getRawMetrics(info)
 
-	if args.All || args.Inventory {
+	e, err := entity(i, &args)
+	fatalIfErr(err)
+
+	if args.HasInventory() {
 		rawInventory := getRawInventory(config, rawMetrics)
-		populateInventory(integration.Inventory, rawInventory)
+		populateInventory(e.Inventory, rawInventory)
 	}
 
-	if args.All || args.Metrics {
+	if args.HasMetrics() {
 		fatalIfErr(metricsErr)
-		ms := integration.NewMetricSet("RedisSample")
+
+		ms := e.NewMetricSet("RedisSample", metric.Attr("hostname", args.Hostname))
```

Comment:
I might be wrong, but since we can monitor multiple instances on the same host (different port) I'll add also the port as an attribute. Otherwise I think the SDK will not be able to distinguish values stored on disk (RATE and DELTA metrics) for those instances @mariomac WDYT?

Reply:
Yep, this is pending to be review following the MMF document. https://docs.google.com/document/d/1WJ9CZTDjHJLDIdwRW4_Bm7KJet9kK_JaT7TlFShbdbc/edit

Good point @cristianciutea 👍

Reply:
I tested this with MySQL and it works, you can add it without problem.

Reply:
You already have it for the `RedisKeyspaceSample` below.